src/tools/perf/rte/mad/perftest_mad.c

```diff
@@ -8,8 +8,8 @@
 #include "config.h"
 #endif
 
-#include "perftest_mad.h"
-#include "perftest.h"
+#include <tools/perf/perftest_context.h>
```

Comment:
need to avoid "tools/perf" prefix since this header file is not really installed in that path on the system

Reply:
using relative path and `#include` with double quotes to make it clear.